frontend/src/components/Gym.js

```diff
@@ -0,0 +1,14 @@
+import React from "react"
+
+function Gym(props) {
+    return (
+        <div className="column" key={props.id}>
```

Comment:
same comment as with Activity, let's remove `column` class from here and use it when the Component is used instead.